xbmc/cores/VideoPlayer/DVDSubtitles/webvtt/WebVTTHandler.cpp

```diff
@@ -236,6 +238,7 @@ bool CWebVTTHandler::Initialize()
   m_overrideStyle = (overrideStyles == KODI::SUBTITLES::OverrideStyles::STYLES_POSITIONS ||
                      overrideStyles == KODI::SUBTITLES::OverrideStyles::STYLES);
 
+  CLog::LogF(LOGINFO, "Initialized WebVTT handler");
```

Comment:
as above not needed
we preempt prioritizing speed of execution than committing writings to logs for superfluous information